BasiliskII.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.haxxed.BasiliskII</id>
```

Comment:
This ID is certainly wrong. If the website is https://www.emaculation.com/doku.php/basilisk_ii, it should me `com.emaculation.basilisk_ii` (or something along this). In general it should match a domain used by the project.

Reply:
Well that's kind of the problem with this project. It is very old, the old project leadership has mostly disappeared, so I'm part of a small band of users trying to revive it to modern standards, loosely based around the emaculation forums. I can certainly see about merging this all into the emaculation/macemu project if required.

Reply:
It should be a reverse DNS of an "owned" domain, so if it's mostly a personal fork, `com.github.username.reponame` is also good.

Reply:
Hello, `emaculation/macemu` maintainer here.  I forked `macemu` from `cebix/macemu` in early 2019 when it appeared that the project was without leadership -- [only 2 merges between Nov 2018 and March 2020](https://github.com/cebix/macemu/pulls?page=1&q=is%3Apr+is%3Aclosed), and (at that time) only the rarely-seen owner had write access.  My short term intention for my fork was to revive the project if at all possible by setting up CI/CD and cherry picking some of the unmerged contributions to the `cebix` repo; my long term intention was to transfer ownership to one or more of those long-time contributors in the absence of the original maintainer.

Instead, within the past year the upstream `cebix/macemu` project seems to have passed the torch to a different maintainer (or maintainers) and resumed development.  Unless & until that team decides that they prefer my namespace to theirs, `cebix/macemu` should be considered the appropriate "upstream" for this package.

Reply:
Okay I'll set the app id to the cebix github.

Reply:
Alright. What should the app id be then? net.cebix.basilisk?